modules/shared_data/src/main/scala/org/elpaca_metagraph/shared_data/combiners/YouTubeCombiner.scala

```diff
@@ -0,0 +1,83 @@
+package org.elpaca_metagraph.shared_data.combiners
+
+import cats.syntax.all._
+import monocle.Monocle.toAppliedFocusOps
+import org.elpaca_metagraph.shared_data.Utils.{epochProgressOneDay, toTokenAmountFormat}
+import org.elpaca_metagraph.shared_data.app.ApplicationConfig
+import org.elpaca_metagraph.shared_data.types.DataUpdates.YouTubeUpdate
+import org.elpaca_metagraph.shared_data.types.States.{DataSource, DataSourceType, YouTubeDataSource}
+import org.elpaca_metagraph.shared_data.types.YouTube.{YouTubeDataSourceAddress, YouTubeRewardInfo}
+import org.tessellation.schema.epoch.EpochProgress
+
+object YouTubeCombiner {
+  private def isNewDay(data: YouTubeRewardInfo, currentEpochProgress: EpochProgress): Boolean =
```

Comment:
This function now belongs to the Utils, so use the function declared there instead of creating a new one